src/ioctls/vcpu.rs

```diff
@@ -1403,6 +1403,12 @@ impl VcpuFd {
         kvm_run.immediate_exit = val;
     }
 
+    /// Sets the `request_interrupt_window` flag on the `kvm_run` struct associated with this vCPU to `val`.
+    pub fn set_kvm_request_interrupt_window(&self, val: u8) {
```

Comment:
Can you also add a test for this, it can be similar to [the one for `set_kvm_immediate_exit`](https://github.com/QuarkContainer/kvm-ioctls/blob/main/src/ioctls/vcpu.rs#L2323)?